src/main/java/jp/co/future/uroborosql/utils/StringFunction.java

```diff
@@ -507,4 +507,53 @@ public String endsWith(final CharSequence text) {
 		}
 	}
 
+	/**
+	 * 加算を行う. 最大値に対して加算した場合はオーバーフローする.<br>
+	 * 引数の型（short/int/longのいづれか）に合わせて計算を行う.
+	 *
+	 * @param num 加算を行う数値
+	 * @return 加算後の数値（オーバーフローした場合は引数の型の最小値になる）
+	 */
+	public Object increment(final Object num) {
+		if (num instanceof Short) {
+			return incrementShort((Short) num);
+		} else if (num instanceof Integer) {
+			return incrementInt((Integer) num);
+		} else if (num instanceof Long) {
+			return incrementLong((Long) num);
+		} else {
+			throw new IllegalArgumentException("Invalid value type. num must be of type short/int/long.");
+		}
+	}
+
+	/**
+	 * short型で加算を行う. 最大値に対して加算した場合はオーバーフローする.
+	 *
+	 * @param num 加算を行う数値
+	 * @return 加算後の数値（オーバーフローした場合はshort型の最小値になる）
+	 */
+	public short incrementShort(final short num) {
```

Comment:
@HidekiSugimoto189 

Since it takes unboxing when called from increment, I think it is better to send it to the wrapper class.

Reply:
OK. Change to wrapper class.